server/routes/auth.route.js

```diff
@@ -19,4 +50,7 @@ router.route('/register')
 router.route('/random-number')
   .get(expressJwt({ secret: config.jwtSecret }), authCtrl.getRandomNumber);
 
+router.route('/facebook/callback')
```

Comment:
@TheHollidayInn you originally said the endpoint should take the name of the social network as a param. Do you advise against explicitly defining each route like this?

Reply:
Nah, I don't feel strongly wither way. Not unless there is a ton of duplicate business logic, which there ins't.